src/main/java/mate/academy/dao/impl/MovieDaoImpl.java

```diff
@@ -39,12 +38,16 @@ public Optional<Movie> get(Long id) {
         try (Session session = HibernateUtil.getSessionFactory().openSession()) {
             return Optional.ofNullable(session.get(Movie.class, id));
         } catch (Exception e) {
-            throw new DataProcessingException("Can't get a movie by id: " + id, e);
+            throw new DataProcessingException("Can't get a movie by id: " + id);
```

Comment:
Include the original exception as a cause in the `DataProcessingException` to provide more context. Use `throw new DataProcessingException("Can't get a movie by id: " + id, e);`.